### PR TITLE
✨ build buttons in components

### LINKED ISSUE #13: build buttons in components

### PR DESCRIPTION
**Pull request by mewiemers**
Fixes #13

add the first two buttons in storybook

### CODE REVIEW

**Review by m-wiemers — APPROVED** [review]
We are building an app. And Melli shows how it works ⛑

File: components/button/Button.module.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.btn {
+  border-radius: 20px;
+  font-size: 1.25rem;
+  background-color: var(--button-color);
+  border: none;
+  font-family: "Gabriola", sans-serif;
+  width: 7rem;
+  height: 2rem;
```
Comment (phlgr) [inline components/button/Button.module.css:8]: Would not set a fixed `height` and `width`, the Button should get it's size by the content it is carrying.

File: components/quizbutton/Quizbutton.module.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  border-radius: 20px;
+  font-size: 1.25rem;
+  background-color: var(--quiz-button-true-color);
+  border: none;
+  font-family: "Gabriola", sans-serif;
+  width: 16rem;
+  height: 4.5rem;
+  box-shadow: 0.01px 7px 16px -2px rgba(0, 0, 0, 0.69),
+    0.01px 7px 16px -2px rgba(0, 0, 0, 0.69);
+}
```
Comment (phlgr) [inline components/quizbutton/Quizbutton.module.css]: You could create a new class standard which combines the standard attributes and remove the standard attributes from the color classes

File: components/quizbutton/Quizbutton.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  color: "blue" | "red" | "green";
+};
+
+export default function QuizButton({
+  label,
+  color,
+  ...props
+}: QuizButtonProps) {
+  return (
+    <button className={styles[color]} {...props}>
```
Comment (phlgr) [inline components/quizbutton/Quizbutton.tsx]:
```suggestion
    <button className={`${styles.standard} ${styles[color]}`} {...props}>
```

**Review by phlgr — CHANGES_REQUESTED** [review]
Gj 🚀

**Pull request merged**